config/locales/pl.yml

```diff
@@ -96,11 +96,8 @@ pl:
       new_event: 'Nowe wydarzenie'
     show:
       users_of_events: 'Lista osób biorących udział w wydarzeniu:'
-    edit:
-      edit_event: 'Popraw informacje o wydarzeniu'
-      links: 
-        show: 'Pokaż zmiany'
-        back: 'Powrót'
-    form:
-      submit: 'Zapisz zmiany'
-      name: 'Nazwa wydarzenia:'
+  profile:
```

Comment:
If you add and change something in one language you need to to the same for the second one.